Tests/IntegrationTest.cs

```diff
@@ -16,34 +16,32 @@ namespace Tests;
 
 public abstract class IntegrationTest : IAsyncLifetime
 {
-    protected readonly ITestOutputHelper _testOutputHelper;
-    protected readonly string _testDisplayName = nameof(AmqpTests);
-    protected readonly TimeSpan _waitSpan = TimeSpan.FromSeconds(5);
+    protected readonly string TestDisplayName = nameof(AmqpTests);
+    protected readonly TimeSpan WaitSpan = TimeSpan.FromSeconds(5);
 
-    protected IConnection? _connection;
-    protected IManagement? _management;
-    protected string _queueName;
-    protected string _exchangeName;
-    protected string _containerId = $"integration-test-{Now}";
+    protected IConnection? Connection;
+    protected IManagement? Management;
+    protected string QueueName;
```

Comment:
Could you please change your IDE to not make this change? In the other .NET clients the style we use for fields is to prefix them with an underscore.

Reply:
Let me check the `.editorconfig` to make sure this is being enforced.

Reply:
Should be the `_` only for private member? btw I will change ! :)

Reply:
>Let me check the .editorconfig to make sure this is being enforced. 

Ok thank you

Reply:
@Gsantomaggio I added enforcement of this rule via 005e072d4d192d23c6ec03317334ed31e45179d8. I'll make sure the other .NET clients do the same. Thanks.